src/ts/src/common/api/index.tsx

```diff
@@ -20,8 +20,8 @@ export const fetchUserData = async (): Promise<
 const SAMPLE_MAP = new Map<string, keyof Sample>([
     ["collection_date", "collectionDate"],
     ["collection_location", "collectionLocation"],
-    ["private_identifier", "privateID"],
-    ["public_identifier", "publicID"],
+    ["private_identifier", "privateId"],
+    ["public_identifier", "publicId"],
     ["upload_date", "uploadDate"],
```

Comment:
do we need to add gisaid to this now too?

Reply:
No, since the gisaid key is not different between the api and the TS type it will transfer automatically :)